README.md

```diff
@@ -1,3 +1,3 @@
 [![Netlify Status](https://api.netlify.com/api/v1/badges/0708f99a-542e-416d-83ce-4ae4f11e9b79/deploy-status)](https://app.netlify.com/sites/centralizedarbitrator/deploys)
 
-Supported netwoks: Mainnet, Kovan, Ropsten
+Supported netwoks: Mainnet, Kovan, Ropsten, Rinkeby.
```

Comment:
Do you also want to list Goerli here?

Reply:
Thanks! Yes, it's actually supported already. I just fixed the readme.